File: UaicLibraryWebSpa/src/app/models/student.ts
import { Faculty } from '../models/faculty';
export class Student {
    public id: number;
    public fullName: string;
    public role: string;
    public readedBooks: number;
    public editedBooks: number;
    public opennedBooks: number;
    public about: string;
    public imageUrl: string;
    public faculties: Array<Faculty>;
    constructor(id: number, fullName: string, readedBooks: number, editedBooks: number, opennedBooks: number, imageUrl: string,
        about: string, faculties: Array<Faculty>) {
        this.id = id;
        this.fullName = fullName;
        this.readedBooks = readedBooks;
        this.editedBooks = editedBooks;
        this.opennedBooks = opennedBooks;
        this.about = about;
        this.imageUrl = imageUrl;
        this.faculties = faculties;
    }
}

File: UaicLibrary.Domain/StudentManagement/Student.cs
using System.Collections.Generic;
using UaicLibrary.Domain.FacultyManagement;

namespace UaicLibrary.Domain.StudentManagement
{
    public class Student
    {
        public int Id { get; set; }
        public string MatricolNumber { get; set; }
        public string Email { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string About { get; set; }
        public string Password { get; set; }
        public string FullName { get; set; }
        public int ReadedBooks { get; set; }
        public int OpennedBooks { get; set; }
        public string ImageUrl { get; set; }
        public int EditedBooks { get; set; }
        public int? Year { get; set; }
        public IList<Faculty> Faculties { get; set; }
    }
}
